src/components/views/avatars/BaseAvatar.tsx

```diff
@@ -109,119 +112,32 @@ const BaseAvatar: React.FC<IProps> = (props) => {
         url,
         urls,
         width = 40,
-        height = 40,
         resizeMethod = "crop", // eslint-disable-line @typescript-eslint/no-unused-vars
-        defaultToInitialLetter = true,
         onClick,
         inputRef,
         className,
+        type = "round",
         altText = _t("Avatar"),
-        ariaLabel = _t("Avatar"),
```

Comment:
Losing this seems bad, this is the aria-label for the button which wraps clickable avatars